crates/iceberg-catalog/migrations/20240911094637_unique_tabular_location.sql

```diff
@@ -2,4 +2,4 @@
 -- select 1 from tabular where '123' like location || '%';
 -- so we drop it and create a new btree index instead that we'll use for exact match queries
 drop index if exists tabular_location_idx;
-create index tabular_namespace_id_location_idx on tabular (namespace_id, location);
+create index tabular_location_idx on tabular (namespace_id, location);
```

Comment:
why the rename? We should not edit existing migrations, this will cause migrations to fail if they are ran against an existing DB.

Reply:
Fixed